diagnostic.md

```diff
@@ -100,6 +125,8 @@ div {
 
 <!-- your answer starts here -->
 
+the .specialDiv div will have a font-family of sans-serif and a color of yellow
+
```

Comment:
font family will actually be cursive because the inline specificity of that styling is always the most powerful